src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+a {
+  display: inline-block;
+  padding: 2em 0;
+  cursor: pointer;
+  line-height: 1em;
+}
+
+ul {
+  list-style: none;
+}
+
+.logo__link {
+  font-size: 0;
+  display: flex;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  height: 60px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  height: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+}
+@media(max-width:1024px) {
+  .nav__list {
+    padding-left: 33px;
+  }
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  width: 37px;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  right: 0;
+}
```

Comment:
it should work same for other links too if I'll add this link

Reply:
<img width="1438" alt="Screenshot 2023-01-25 at 22 43 46" src="https://user-images.githubusercontent.com/84275496/214686047-eb233584-5da5-44b2-83a5-3d2d02039104.png">